docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -0,0 +1,169 @@
+---
+id: index
+title: "Zeebe Gateway"
+sidebar_label: "Overview"
+description: "The Zeebe Gateway is a component of the Zeebe cluster."
+---
+
+## What is the Zeebe Gateway?
+
+The Zeebe Gateway is a component of the Zeebe cluster; it can be considered the contact point for the Zeebe cluster which allows Zeebe clients to communicate with Zeebe brokers inside a Zeebe cluster. For more information about the Zeebe broker, visit our [additional documentation](../../components/zeebe/technical-concepts/architecture.md#brokers).
+
+To summarize, the Zeebe broker is the main part of the Zeebe cluster, which does all the heavy work like processing, replicating, exporting, and everything based on partitions. The Zeebe Gateway acts as a load balancer and router between Zeebe’s processing partitions.
+
+![Zeebe gateway overview](assets/zeebe-gateway-overview.png)
+
+To interact with the Zeebe cluster, the Zeebe client sends a command as a gRPC message to the Zeebe Gateway (to port `26500` by default). Given the gateway supports gRPC, the user can use several clients in different languages to interact with the Zeebe cluster. For more information, read our [overview](../../apis-clients/overview.md).
+
+When the Zeebe Gateway receives a valid gRPC message, it is translated to an internal binary format and forwarded to one of the partition leaders inside the Zeebe cluster. The command type and values can determine to which partition the command is forwarded.
+
+For example, creating a new process instance is sent in a round-robin fashion to the different partitions. If the command relates to an existing process instance, the command must be sent to the same partition where it was first created (determined by the key).
+
+To determine the current leader for the corresponding partition, the gateway must maintain the topology of the Zeebe cluster. The gateway(s) and broker(s) form a cluster using gossip to distribute information.
+
+## Why do we have it and what problems does it solve?
+
+The Zeebe Gateway protects the brokers from external sources. It allows the creation of a demilitarized zone ([DMZ](https://en.wikipedia.org/wiki/DMZ_(computing))) and the Zeebe Gateway is the only contact point.
+
+The Zeebe Gateway also allows you to easily create clients in your language of choice while keeping the client implementation as thin as possible. There are already several client implementations available, officially-supported, and community-maintained. Check the list [here](../../apis-clients/overview.md).
+
+The gateway can be run and scaled independently of the brokers, which allows separating the concerns of the applications.
```

Comment:
What do you mean by separating the concerns of the applications?

Reply:
I meant with that actually that the gateway just does the translating of the messages and distributes the incoming messages to the right partition leaders and the broker just accepts and processes them. Instead of one application doing everything we are able to split the concerns, does it make sense? :)

Reply:
Adjusted to `The gateway can be run and scaled independently of the brokers, which means it translates the messages, distributes them to the correct partition leaders, and separates the concerns of the applications.` in latest commit. Thanks!